roles/haproxy/README.md

```diff
@@ -85,6 +85,25 @@ certs: A list of certificates that you use for https. That list contains these p
  key_content: You can add your https key in this parameter, and add it to your secrets. See environments/vm/secrets/vm.yml for an example
  crt_name: The certificate is placed in your environments directory: In files/certs/. The filename should match "crt_name"
 
+## Rate limiting
+The frontend has a limit that is configurable for the amount of requests per 10s. The rationale is that you want to stop sudden bursts of requests fired by clients, and you want to be quick enough to block it, before your backends are saturated. The default is 1000 requests per 10s. In our logs, we observed some NAT gateways of larger instutitions that were able to generate more than 500 legitimate requests per 10s. All rates above 1000 per 10s seemed to be coming from bots, scrapers, or misbehaving clients.
+If you want to exclude ip's from the limit, you can add them to your configuration under ```haproxy_allowlistips```. Suppose you want to put the ip address 1.1.1.1 and the ip addres 2.2.2.2 to the allowlist, you can do so by putting the following in your environment:
```

Comment:
IPs? Or cidrs?

Reply:
both, I've updated the documentation to reflect that

Reply:
I don’t see that change yet?

Reply:
Somehow it got in my Git stash. I've committed it now!